### ISSUE TITLE
possible for you share a rough timeline for sharing training code?

### ISSUE THREAD

**Issue by exllama-fan** [yl4579/StyleTTS2#2]
Have been tinkering with StyleTTS to achieve a good ASMR TTS voice.
sharing a tortoise sample here
https://drive.google.com/file/d/1OfZnURiaAEW0O_B8RVozt1M4Uadl-rEV/view?usp=sharing

haven't had good experience finetuning STyleTTS so far. Would you like to tell when the next version will be out?

**Comment by yl4579** [comment]
It will be available by the end of this month. As for ASMR TTS, it probably needs more than the current framework of StyleTTS (or StyleTTS 2), because it is mostly unvoiced whisper (so F0 and energy do not make too much sense here). You may want to look for papers working on whisper speech synthesis and see if you can bring some ideas from there.

**Comment by exllama-fan** [comment]
What I infer is that duration accuracy is crucial to pick the speech style, it does not need to pick the exact whispered speech.

my experiments with finetuning styleTTS (with PL_BERT) using different datasets show that using CE loss, duration loss attains a value of 0.2 (starting from 1.2) during second stage training while distorting all other losses. Although I haven't really tried inferring all of them, but my desire is to match the speech duration with the ground truth, voice can be changed through a pipeline, since the inference speed of styleTTS is so fast.

with CE loss off, the best mel_loss I got was ~.23 during first stage. I keep changing lr from .00005 to .0001 as per the dataset size. Sharing all this info because I would want to meet the ideal scenario for the kind of speech I am looking to generate.

Also, I am keeping datasets with number of clips ranging for 1000-2000 for the purpose of finetuning
